cl/search/templates/advanced.html

```diff
@@ -601,7 +601,7 @@ <h3>Background</h3>
                 <p>This work has been possible thanks to <a href="https://free.law/thanks/">financial and volunteer support</a> from a growing list of people and organizations. If this work is valuable to you, your research, or your organization, please donate so our work can continue and thrive.
                 </p>
                 <p>
-                    <a href="{% url "donate" %}?referrer=opinion-adv-search"
+                    <a href="https://donate.free.law/forms/supportflp"
```

Comment:
Can you make this say "please donate or become a member so...", and then add a second button next to the first one for [Join Free.law] ?

Reply:
Yes of course!